src/classlibnative/inc/nlsinfo.h

```diff
@@ -145,53 +86,7 @@ class COMNlsInfo {
 
     static void QCALLTYPE nativeNormalizationInitNormalization(int NormForm, BYTE* pTableData);
```

Comment:
my changes was mainly to kill the duplication in the globalization code (i.e. get rid of the files under corefx folder). normalization stuff is not part of this, it is  used by string/System.Text. we can do your suggestion later.